spec/rubocop/cop/rspec/empty_line_after_hook_spec.rb

```diff
@@ -19,6 +19,126 @@
     RUBY
   end
 
+  it 'allows multiple one-liner blocks' do
```

Comment:
Can you please wrap examples that cover the default style in a context, too? I have an impression that this is quite common in our code base (see e.g. https://github.com/rubocop/rubocop-rspec/blob/a8b16a67b958acb0972e288fa191e8a65719da3e/spec/rubocop/cop/rspec/be_nil_spec.rb#L8).